### ISSUE TITLE
Seems like an interesting project, any ETA for a first release?

### ISSUE THREAD

**Issue by viztastic** [bkinsey808/angular-offline-first-graphql#1]
Keen to learn how others implement JWT and GraphQL, looking forward to your first release @bkinsey808

**Comment by bkinsey808** [comment]
Thanks. No ETA, I got distracted working on my own version of ngxerrors. I'd like to bring focus back on this project soon. I also noticed your project, looks interesting: https://github.com/viztastic/magnet maybe we can collaborate.

**Comment by viztastic** [comment]
Certainly, would be happy to @bkinsey808 I'm actually using MAGNET on a real project and looking to bring across the learnings from where that project ends up across to the open source version I'm working on.

Good luck and be in touch.
